configure

```diff
@@ -50,5 +50,22 @@ fi
 
 version=`pkg-config --modversion zlib`
 echo "zlib ${version} is available ok"
-exit 0
 
+# Find R compilers
+CC=`${R_HOME}/bin/R CMD config CC`
+CFLAGS=`${R_HOME}/bin/R CMD config CFLAGS`
+CPPFLAGS=`${R_HOME}/bin/R CMD config CPPFLAGS`
+
+# Test if we have a OPENMP compatible compiler
+echo "#include <omp.h>\nint main () { return omp_get_num_threads (); }" | ${CC} ${CPPFLAGS} ${PKG_CFLAGS} ${CFLAGS} ${SHLIB_OPENMP_CFLAGS} -E -xc - >/dev/null 2>&1 || R_NO_OPENMP=1;
```

Comment:
Here we test a simple OpenMP program with what is in `SHLIB_OPENMP_CFLAGS` to verify that our current compiler supports OpenMP.